src/main/java/interface_adapter/search/SearchController.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.search;
+
+import use_case.search.SearchInputBoundary;
+
+/**
+ * Handles user input from the UI and forwarding it to the search interactor.
+ */
+
+public class SearchController {
+    private final SearchInputBoundary interactor;
+
+    public SearchController(SearchInputBoundary interactor) {
+        this.interactor = interactor;
+    }
+
+    public void performSearch(String keyword) {
+        interactor.searchPostsByTitle(keyword);
+    }
+}
```

Comment:
The SearchController class in the interface_adapter.search package is designed to function as part of the Clean Architecture model, particularly aligning with the Interface Adapter layer.